components/automate-ui/src/app/helpers/auth/sorter.ts

```diff
@@ -0,0 +1,39 @@
+export class ChefSorters {
+    // ***********
+    // naturalSort takes in an Array of Objects and returns an Array of Objects
+    // sorted 'naturally' by the property chosen.
+    // @input: Array of Objects
+    // @property: string or an Array of strings, max length is 2 properties
+    //            when using an Array, sort by first property then by second property
+    //
+    // example use with single prop:  ChefSorters.naturalSort(users, 'name');
+    // example use with dual prop:    ChefSorters.naturalSort(users, ['name', 'id']);
+    // ***********
+
+    // tslint:disable-next-line: max-line-length
+    public static naturalSort(input: Array<any>, property: Array<string> | string): Array<any> {
```

Comment:
Prefer to use `tslint` decorators only when it is a true burden; here you can just add a carriage return, e.g.:
```suggestion
    public static naturalSort(
      input: Array<any>, property: Array<string> | string): Array<any> {
```